src/14.1/com/intellij/plugins/haxe/runner/debugger/HaxeDebugRunner.java

```diff
@@ -760,8 +760,8 @@ public StackFrame(Project project, Module module,
         mFileName = (String)frameList.params.__a[4];
         mLineNumber = (((Integer)frameList.params.__a[5]).intValue());
         mClassAndFunctionName =
-          ((String)frameList.params.__a[2] + "." +
-           (String)frameList.params.__a[3]);
+          (frameList.params.__a[2] + "." +
+           frameList.params.__a[3]);
```

Comment:
Don't know for sure, but it seems maybe I used some code cleanup feature of IDEA, that says that it is redundant cast